src/Components/RosToolbar/RosToolbar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarGroup, ToolbarContent, TextInput } from '@patternfly/react-core';
+import { TablePagination } from '../Pagination/Pagination';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+//import propTypes from 'prop-types';
+
+export class RosToolbar extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <React.Fragment>
```

Comment:
No need to wrap in `React.Fragment` as the component returns just `Toolbar` component with inner components.